index.html

```diff
@@ -73,7 +73,6 @@ <h1>Developed by <a href="https://www.github.com/FIRSTMap/firstmap.github.io/gra
 
     <!-- Javascript Scripts -->
     <script src="data/teamInfo.js"></script>
```

Comment:
Yeah, unfortunately you've got a lot of conflicts since I mostly rewrote the app today.